app/validation/validator.py

```diff
@@ -281,6 +282,16 @@ def _validate_answer_actions(self, answer):
 
         return errors
 
+    def _ensure_answer_labels_and_values_match(self, answer):
+        errors = []
+        if 'options' in answer:
+            for options in answer['options']:
+                if 'placeholders' not in options['label']:
+                    if options['label'] != options['value']:
+                        errors.append(
+                            (self._error_message(f'`Found mismatching answer value(s) and label(s) for {answer["id"]}`')))
+        return errors
```

Comment:
Worth reducing some nesting. We also seem to already use the pattern of `isinstance` for option labels.
```py
    def _ensure_answer_labels_and_values_match(self, answer):
        errors = []
        for option in answer.get('options', []):
            if isinstance(option['label'], str) and option['label'] != option['value']:
                errors.append(
                    (self._error_message(f'`Found mismatching answer value(s) and label(s) for {answer["id"]}`')))

        return errors
```

Reply:
Fixed this